packages/entando-apimanager/modules/state/api/reducer.js

```diff
@@ -3,7 +3,6 @@ import { SET_API } from './types';
 const initialState = {
   useMocks: true,
   domain: null,
```

Comment:
I think domain by default should be an empty string rather than null. Otherwise the case where the domain is not set in the environment neither in the request property will generate an error as `${domain}` will resolve to 'null'

Reply:
Add a unit test for this too

Reply:
updated